specification/columns/effectivecost.md

```diff
@@ -24,7 +24,7 @@ Effective Cost
 
 ## Description
 
-Cost inclusive of the impacts of all reduced rates and discounts, augmented with the *amortization* of relevant purchases (one-time or recurring) paid to cover future eligible *charges*.
+Cost inclusive of the impacts of all reduced rates and discounts, augmented with the *amortization* of relevant purchases (one-time or recurring) paid to cover future eligible *charges*, with the exception that, in the case of a purchase charge paid to cover future eligible charges, it amounts to 0.
```

Comment:
@ijurica are you trying to say effective cost must be the amortization of purchases to cover future charges but must not include the upfront cost of the purchase charge?

Reply:
I like the suggestion, but I see this as a clarification statement rather than an "exception".

Maybe just make it a new sentence like:
"Cost inclusive of the impacts of all reduced rates and discounts, augmented with the *amortization* of relevant purchases (one-time or recurring) paid to cover future eligible *charges*. In the case of a purchase charge paid to cover future eligible charges, EffectiveCost amounts to 0."